Modules/selectmodule.c

```diff
@@ -594,7 +594,7 @@ poll_poll(pollObject *self, PyObject *args)
                 poll_result = 0;
                 break;
             }
-            ms = _PyTime_AsMilliseconds(timeout, _PyTime_ROUND_CEILING);
+            ms = _PyTime_AsMilliseconds(timeout, _PyTime_ROUND_UP);
```

Comment:
Since `timeout` >= 0, there is no difference between `_PyTime_ROUND_CEILING` and `_PyTime_ROUND_UP` here.

Reply:
Done in 5c96cd6